src/main/java/com/yahoo/bullet/record/avro/UntypedAvroBulletRecord.java

```diff
@@ -42,6 +42,12 @@ protected UntypedAvroBulletRecord rawSet(String field, Serializable object) {
         return this;
     }
 
+    @Override
+    protected Map<String, Serializable> getRawDataMap() {
+        data.forceReadData();
+        return (Map) data.getData();
```

Comment:
Cast as Map<String, Serializable> just to make it clear. And in TypedAvro

Reply:
Unfortunately can't cast data from Map<String, Object> to Map<String, Serializable>

Current getRawDataMap in the avro records uses an iterator that casts each value to Serializable first. This bypasses that, but I think our (safe?) assumption is that everything should be serializable. (Though, it's not actually necessary for the values to be serializable given that we only use this for json results)

Reply:
Actually, Avro can contain types (like the type for String) that aren't Serializable. This is why Bullet DSL needs that adapter for Avro.

Reply:
Mm, are you ok with this though? The difference is that originally it'd throw a classcastexception if there was a non-serializable object in there, and now it wouldn't.

Reply:
Sure we can assume everything that went into the record was Serializable because it used our sets